### PR TITLE
dolt sql shell slash redux

### PR DESCRIPTION
**Pull request by macneale4**

### CODE REVIEW

**Comment by coffeegoddd** [comment]
@macneale4 __DOLT__
 <table>
<TR><TH>comparing_percentages</TH>
</TR>
<TR><TD>100.000000 to 100.000000</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>result</TH>
<TH>total</TH>
</TR>
<TR><TD>63ae991</TD>
<TD>ok</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>total_tests</TH>
</TR>
<TR><TD>63ae991</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>correctness_percentage</TH>
</TR>
<TR><TD>100.0</TD>
</TR>
</table>

File: go/cmd/dolt/commands/sql.go [hunk 1]
```diff
@@ -746,11 +751,13 @@ func execShell(sqlCtx *sql.Context, qryist cli.Queryist, format engine.PrintResu
...
 		query := c.Args[0]
-		if len(strings.TrimSpace(query)) == 0 {
+		query = strings.TrimSpace(query)
+		if len(query) == 0 {
 			return
 		}
 
-		closureFormat := format
+		var nextPrompt string
+		var multiPrompt string
```
Comment (jennifersp) [inline go/cmd/dolt/commands/sql.go:760]: These should go back to where they were closer to where they are used.

File: go/cmd/dolt/commands/sql.go [hunk 2]
```diff
@@ -773,43 +781,48 @@ func execShell(sqlCtx *sql.Context, qryist cli.Queryist, format engine.PrintResu
...
-				shell.Println(verr.Verbose())
-			} else if rowIter != nil {
-				switch closureFormat {
-				case engine.FormatTabular, engine.FormatVertical:
-					err = engine.PrettyPrintResultsExtended(sqlCtx, closureFormat, sqlSch, rowIter)
-				default:
-					err = engine.PrettyPrintResults(sqlCtx, closureFormat, sqlSch, rowIter)
-				}
-
+			_, foundCmd := findSlashCmd(query)
```
Comment (jennifersp) [inline go/cmd/dolt/commands/sql.go:793]: I think you can use the `cli.Command` returned from this method passed into `handleSlashCommand` method so you don't have to call `findSlashCmd` twice if `foundCmd` is true.

**Review by jennifersp — COMMENTED** [review]
A small suggestion

**Review by jennifersp — APPROVED** [review]
LGTM!

**Comment by coffeegoddd** [comment]
@macneale4 __DOLT__
 <table>
<TR><TH>comparing_percentages</TH>
</TR>
<TR><TD>100.000000 to 100.000000</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>result</TH>
<TH>total</TH>
</TR>
<TR><TD>bd210d8</TD>
<TD>ok</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>total_tests</TH>
</TR>
<TR><TD>bd210d8</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>correctness_percentage</TH>
</TR>
<TR><TD>100.0</TD>
</TR>
</table>

**Comment by coffeegoddd** [comment]
@macneale4 __DOLT__
 <table>
<TR><TH>comparing_percentages</TH>
</TR>
<TR><TD>100.000000 to 100.000000</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>result</TH>
<TH>total</TH>
</TR>
<TR><TD>60372fa</TD>
<TD>ok</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>total_tests</TH>
</TR>
<TR><TD>60372fa</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>correctness_percentage</TH>
</TR>
<TR><TD>100.0</TD>
</TR>
</table>

**Pull request merged**